test/Exercism.Analyzers.CSharp.IntegrationTests/Solutions/WeighingMachine/Approve/WeighingMachine.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+
+class WeighingMachine
+{
+    private double _weight;
+    private int _precision;
+
+    public WeighingMachine(int precision)
+    {
+        Precision = precision;
+    }
+
+    public int Precision
+    {
+        get { return _precision; }
+        private set
+        {
+            _precision = value;
+        }
+    }
```

Comment:
I think we'd want people to do:

```suggestion
    public int Precision { get; private set; }
```

It might be good to add a rule for that in the analyzer too.

Reply:
Rule already exist, looks like test wasn't running before

Reply:
Do we have a test case that verifies that a comment is made when the student uses:

```csharp
public int Precision
    {
        get { return _precision; }
        private set
        {
            _precision = value;
        }
    }
```
?

Reply:
It checks that autoproperty is used

Reply:
https://github.com/exercism/csharp-analyzer/pull/109/files#diff-91639737ce24cbf45cff1d2e6fed7e8ca2ef944bdc33cb5e2549bcb85f3786e2R24